app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,20 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+ 
+  def perform(initiator, benchmark_type, benchmark_result_type, category)
+    commit_objects = BenchmarkRun.where( initiator: initiator, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).limit(1000)
```

Comment:
Right now we only have one server that is running the benchmarks, benchmark results will come in orderly. However, if this job fails for whatever reason and we need to retry it. The benchmark_runs that you fetch might be a totally different set of data.